### PR TITLE
Aligning with changes in Structure_Engine

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### NOTE: Depends on 

https://github.com/BHoM/BHoM_Engine/pull/2146/
   
### Issues addressed by this PR

https://github.com/BHoM/BHoM_Engine/issues/2135

Aligning with methods being removed from Structure_Engine.

This is only adding a reference and changing a to target another pre-existing method, so can safely be merged before others.

### CODE REVIEW

File: Lusas_Engine/Lusas_Engine.csproj [hunk 1]
```diff
@@ -86,6 +86,9 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Analytical_Engine">
+      <HintPath>C:\ProgramData\BHoM\Assemblies\Analytical_Engine.dll</HintPath>
```
Comment (peterjamesnugent) [inline Lusas_Engine/Lusas_Engine.csproj:90]: `      <Private>False</Private>`

**Review by peterjamesnugent — CHANGES_REQUESTED** [review]
Add Private field for consistency

**Comment by IsakNaslundBh** [comment]
@BHoMBot check project compliance

**Comment by IsakNaslundBh** [comment]
@BHoMBot fix project file ref. 1395807062

**Review by peterjamesnugent — APPROVED** [review]
LGTM.

**Pull request merged**